### ISSUE TITLE
Unnamed local variables are misidentified as backing properties

### ISSUE THREAD

**Issue by viluon** [pinterest/ktlint#3205]
## Expected Behavior

Usage of unnamed local variables, [recommended for suppressing unused return value warnings](https://kotlinlang.org/docs/whatsnew-eap.html#unused-return-value-checker), would be accepted by ktlint.

```kt
fun ktlintIssueRepro() {
    // use of '-XXLanguage:+UnnamedLocalVariables' or Kotlin 2.3 unused return value checker
    val _ = "ignored"
}
```

## Observed Behavior

There are false positives for backing property naming:
```
> Task :ktlintTestSourceSetCheck FAILED
KtlintIssueRepro.kt:3:9 Backing property should start with underscore followed by lower camel case (cannot be auto-corrected)
KtlintIssueRepro.kt:3:9 Backing property not allowed when 'private' modifier is missing (cannot be auto-corrected)
KtlintIssueRepro.kt:3:9 Backing property is only allowed when a matching property or function exists (cannot be auto-corrected)

FAILURE: Build failed with an exception.
```

Very similar to
- #2779

## Steps to Reproduce
Run ktlint 1.7.1 on the above code snippet.

## Your Environment

* Version of ktlint used: 1.7.1
* Relevant parts of the `.editorconfig` settings: `ktlint_code_style = intellij_idea`
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): `org.jlleitschuh.gradle.ktlint`, version 14.0.1
* Version of Gradle used (if applicable): Gradle 9.2.1
* Operating System and version: NixOS 25.05, Linux 6.17.9

**Comment by paul-dingemans** [comment]
This is fixed by #3166 in ktlint `1.8.0`.

**Comment by viluon** [comment]
My mistake, I didn't check the release notes. Thank you for the swift response @paul-dingemans, apologies for adding to the maintenance burden.
